scripts/ChangesSinceLastRelease.txt

```diff
@@ -954,3 +954,87 @@ Windows.Win32.Devices.DeviceAndDriverInstallation.SETUP_DI_BUILD_DRIVER_DRIVER_T
+Windows.Win32.Graphics.Dxgi.Apis.DXGI_SHARED_RESOURCE_WRITE removed
+Windows.Win32.Graphics.Dxgi.DXGI_CREATE_FACTORY added
+Windows.Win32.Graphics.Dxgi.DXGI_CREATE_FACTORY.DXGI_CREATE_FACTORY_DEBUG added
+Windows.Win32.Graphics.Dxgi.DXGI_MAP added
+Windows.Win32.Graphics.Dxgi.DXGI_MAP.DXGI_MAP_DISCARD added
+Windows.Win32.Graphics.Dxgi.DXGI_MAP.DXGI_MAP_READ added
+Windows.Win32.Graphics.Dxgi.DXGI_MAP.DXGI_MAP_WRITE added
+Windows.Win32.Graphics.Dxgi.DXGI_MWA added
+Windows.Win32.Graphics.Dxgi.DXGI_MWA.DXGI_MWA_NO_ALT_ENTER added
+Windows.Win32.Graphics.Dxgi.DXGI_MWA.DXGI_MWA_NO_PRINT_SCREEN added
+Windows.Win32.Graphics.Dxgi.DXGI_MWA.DXGI_MWA_NO_WINDOW_CHANGES added
+Windows.Win32.Graphics.Dxgi.DXGI_MWA.DXGI_MWA_VALID added
+Windows.Win32.Graphics.Dxgi.DXGI_PRESENT added
+Windows.Win32.Graphics.Dxgi.DXGI_PRESENT.DXGI_PRESENT_ALLOW_TEARING added
+Windows.Win32.Graphics.Dxgi.DXGI_PRESENT.DXGI_PRESENT_DO_NOT_SEQUENCE added
+Windows.Win32.Graphics.Dxgi.DXGI_PRESENT.DXGI_PRESENT_DO_NOT_WAIT added
+Windows.Win32.Graphics.Dxgi.DXGI_PRESENT.DXGI_PRESENT_RESTART added
+Windows.Win32.Graphics.Dxgi.DXGI_PRESENT.DXGI_PRESENT_RESTRICT_TO_OUTPUT added
+Windows.Win32.Graphics.Dxgi.DXGI_PRESENT.DXGI_PRESENT_STEREO_PREFER_RIGHT added
+Windows.Win32.Graphics.Dxgi.DXGI_PRESENT.DXGI_PRESENT_STEREO_TEMPORARY_MONO added
+Windows.Win32.Graphics.Dxgi.DXGI_PRESENT.DXGI_PRESENT_TEST added
+Windows.Win32.Graphics.Dxgi.DXGI_PRESENT.DXGI_PRESENT_USE_DURATION added
+Windows.Win32.Graphics.Dxgi.DXGI_RESOURCE_PRIORITY added
+Windows.Win32.Graphics.Dxgi.DXGI_RESOURCE_PRIORITY.DXGI_RESOURCE_PRIORITY_HIGH added
+Windows.Win32.Graphics.Dxgi.DXGI_RESOURCE_PRIORITY.DXGI_RESOURCE_PRIORITY_LOW added
+Windows.Win32.Graphics.Dxgi.DXGI_RESOURCE_PRIORITY.DXGI_RESOURCE_PRIORITY_MAXIMUM added
+Windows.Win32.Graphics.Dxgi.DXGI_RESOURCE_PRIORITY.DXGI_RESOURCE_PRIORITY_MINIMUM added
+Windows.Win32.Graphics.Dxgi.DXGI_RESOURCE_PRIORITY.DXGI_RESOURCE_PRIORITY_NORMAL added
+Windows.Win32.Graphics.Dxgi.IDXGIDecodeSwapChain.PresentBuffer : Flags...UInt32 => DXGI_PRESENT
+Windows.Win32.Graphics.Dxgi.IDXGIFactory.MakeWindowAssociation : Flags...UInt32 => DXGI_MWA
+Windows.Win32.Graphics.Dxgi.IDXGIFactory3.GetCreationFlags : return...UInt32 => DXGI_CREATE_FACTORY
+Windows.Win32.Graphics.Dxgi.IDXGIResource.GetEvictionPriority : pEvictionPriority...UInt32* => DXGI_RESOURCE_PRIORITY*
+Windows.Win32.Graphics.Dxgi.IDXGIResource.SetEvictionPriority : EvictionPriority...UInt32 => DXGI_RESOURCE_PRIORITY
+Windows.Win32.Graphics.Dxgi.IDXGISurface.Map : MapFlags...UInt32 => DXGI_MAP
+Windows.Win32.Graphics.Dxgi.IDXGISwapChain.Present : Flags...UInt32 => DXGI_PRESENT
+Windows.Win32.Graphics.Dxgi.IDXGISwapChain1.Present1 : PresentFlags...UInt32 => DXGI_PRESENT
+Windows.Win32.System.SystemServices.DXGI_SHARED_RESOURCE_RW added
+Windows.Win32.System.SystemServices.DXGI_SHARED_RESOURCE_RW.DXGI_SHARED_RESOURCE_READ added
+Windows.Win32.System.SystemServices.DXGI_SHARED_RESOURCE_RW.DXGI_SHARED_RESOURCE_WRITE added

```

Comment:
Should I set a `namespace` in `enums.json` for this one? I don't see the constants being used anywhere (in favour of the `_OFFER_` variant) and it's ending up in the `SystemServices` namespace...

Reply:
Yep, how does this sound?

```json
"namespace": "Windows.Win32.Graphics.Dxgi",
```

Reply:
Marked this as resolved already because that's exactly what I ended up pushing a week ago.

All the other questions remain open though.